models.py

```diff
@@ -639,10 +646,9 @@ def margin(log_probs, dim = 1):
 def compute_capacity(model, scale = 1):
 	return sum(map(torch.numel, model.parameters())) / scale
 
-
-def normalize_signal(signal, dim = -1, eps = 1e-5):
-	return signal / (signal.abs().max(dim = dim, keepdim = True).values + eps) if signal.numel() > 0 else signal
-
+def normalize_signal(signal, dim = -1, eps = 1e-5, debug_short_long_records_normalize_signal_multiplier  = 1.0):
```

Comment:
naming nit: I propose to rename `debug_short_long_records_normalize_signal_multiplier` -> `denom_multiplier`. It is clear what it does and does not rise many questions, so a short name here would do.